lib/ansible/modules/cloud/azure/azure_rm_aks_facts.py

```diff
@@ -14,14 +14,14 @@
 
 DOCUMENTATION = '''
 ---
-module: azure_rm_aks_facts
+module: azure_rm_aks_info
```

Comment:
you still need to do 2 things:
(1) rename this file
(2) create a link called _azure_rm_aks_facts.py -> azure_rm_aks_info.py
(3) update integration test here: https://github.com/ansible/ansible/blob/devel/test/integration/targets/azure_rm_aks/tasks/main.yml